File: rdk/template/runtime/java8/src/main/java/com/rdk/RuleCode.java
package com.rdk;

import java.io.IOException;
import com.fasterxml.jackson.core.JsonProcessingException;
import com.fasterxml.jackson.databind.JsonNode;
import com.amazonaws.services.config.model.*;

import com.amazonaws.services.lambda.runtime.Context;
import com.amazonaws.services.lambda.runtime.LambdaLogger;


public class RuleCode {

  public static ComplianceType evaluateCompliance(JsonNode invokingEvent, JsonNode ruleParameters, Context context) throws JsonProcessingException,
          IOException {
      LambdaLogger logger = context.getLogger();
      logger.log("Beginning Custom Config Rule Evaluation");

      /*
      YOUR CODE GOES HERE!
      */

      return ComplianceType.NON_COMPLIANT;
  }
}

File: rdk/template/runtime/dotnetcore1.0/.rdk/runtime/dotnetcore2.0/RuleCode.cs
using System;
using System.Collections.Generic;
using System.Text;

using Amazon.ConfigService.Model;
using Amazon.ConfigService;
using Amazon.Lambda.Core;
using Amazon.Lambda.Model;
using Amazon.Lambda.ConfigEvents;
using Newtonsoft.Json.Linq;

namespace Rdk
{
    class RuleCode
    {
        public static ComplianceType EvaluateCompliance(JObject invokingEvent, JObject ruleParameters, ILambdaContext context)
        {
            context.Logger.LogLine("Beginning Custom Config Rule Evaluation");

            /*
            YOUR CODE GOES HERE!
            */

            return ComplianceType.NON_COMPLIANT;
        }
    }
}
